src/Memory.ts

```diff
@@ -1,4 +1,78 @@
 
+/**
+ * Memory implements a simple memory model
+ * for the ethereum virtual machine.
+ */
 export class Memory {
 
+    // TODO replace to Buffer
+    _store: number[];

```

Comment:
maybe use generic and multiple implementations of Memory? (for Array and for Buffer)